iOS Example Frame/iOS Example Frame/MainViewController.swift

```diff
@@ -22,6 +22,11 @@ class MainViewController: UIViewController, CardViewControllerDelegate, ThreedsW
     // Step1 : create instance of CheckoutAPIService
     let checkoutAPIService = Frames.CheckoutAPIService(publicKey: "pk_test_6e40a700-d563-43cd-89d0-f9bb17d35e73",
                                                        environment: .sandbox)
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        customizeNavigationBarAppearance()
```

Comment:
Since you are calling customizeNavigationBarAppearance()  in example app my question is. Do we expect our merchants to call customizeNavigationBarAppearance() on their app ? because I don't expect them to call this method customizeNavigationBarAppearance() as they might have a different style.

Reply:
Merchant may not call it. it's a fix for sample app project.